NuRadioReco/autodoc/pages/detector_database_fields.rst

```diff
@@ -21,15 +21,84 @@ For dipoles, it can point in any direction that is perpendicular to the first ve
 
 The vectors are each described by two angles:
 
-- orientation_theta: boresight direction (zenith angle, 0deg is the zenith, 180deg is straight down)
-- orientation_phi: boresight direction (azimuth angle counting from East counterclockwise)
-- rotation_theta: rotation of the antenna, is perpendicular to 'orientation', for LPDAs: vector in plane of tines pointing away from connector
-- rotation_phi: rotation of the antenna, is perpendicular to 'orientation', for LPDAs: vector in plane of tines pointing away from connector
+- orientation_theta: orientation of the antenna, as a zenith angle (0deg is the zenith, 180deg is straight down); for LPDA: outward along boresight; for dipoles: upward along axis of azimuthal symmetry
+- orientation_phi: orientation of the antenna, as an azimuth angle (counting from East counterclockwise); for LPDA: outward along boresight; for dipoles: upward along axis of azimuthal symmetry
+- rotation_theta: rotation of the antenna, is perpendicular to 'orientation', for LPDAs: vector perpendicular to the plane containing the the tines
+- rotation_phi: rotation of the antenna, is perpendicular to 'orientation', for LPDAs: vector perpendicular to the plane containing the the tines
+
+The user must specify *both* the orientation vector and the rotation vector. 
+
+Here are a few examples.
+
+1. For an LPDA, the following points the LPDA straight down, with tines aligned with
+(parallel to) the x-axis, and therefore pointing eastward.
```

Comment:
you mean "pointing northwards" right? The x-axis is along the East-West line, so the vector perpendicular to it is pointing Northwards.

Reply:
Right, agree. So, the example that I tried to write has the rotation vector along the North/South line, but has the tines along the East/West line. Did I get that wrong?

Reply:
No that is correct. Maybe I misunderstood your comment. I though you meant with eastward the direction of the `orientation` vector. Maybe you can rephrase it to 
"1. For an LPDA, the following points the LPDA straight down, with tines aligned with
(parallel to) the x-axis (i.e. the east-west direction), and therefore the orientation vector points northwards."

Reply:
Yeah, I was trying to talk about the tines, but that tweak adds further clarity. I will make that change!